codegen/src/main/java/io/improbable/keanu/codegen/python/VertexProcessor.java

```diff
@@ -42,18 +48,26 @@ static void process(String generatedDir) {
         Map<String, Object> root = new HashMap<>();
         List<Import> imports = new ArrayList<>();
         List<PythonConstructor> pythonConstructors = new ArrayList<>();
+        List<String> exportedMethodsList = new ArrayList<>();
 
         root.put("imports", imports);
         root.put("constructors", pythonConstructors);
-
+        Map<String, DocString> nameToDocStringMap = KeanuProjectDoclet.getDocStringsFromFile();
         for (Constructor constructor : constructors) {
             String javaClass = constructor.getDeclaringClass().getSimpleName();
+            String qualifiedName = constructor.getName();
+            DocString docString = nameToDocStringMap.get(qualifiedName);
+
             String[] pythonParameters = reflections.getConstructorParamNames(constructor).stream().map(
                 parameter -> CaseFormat.UPPER_CAMEL.to(CaseFormat.LOWER_UNDERSCORE, parameter)).toArray(String[]::new);
 
             imports.add(new Import(constructor.getDeclaringClass().getCanonicalName()));
-            pythonConstructors.add(new PythonConstructor(javaClass, toPythonClass(javaClass), String.join(", ", pythonParameters)));
+            PythonConstructor pythonConstructor = new PythonConstructor(javaClass, toPythonClass(javaClass), String.join(", ", pythonParameters), docString.getAsString());
+            pythonConstructors.add(pythonConstructor);
+            exportedMethodsList.add(pythonConstructor.pythonClass);
         }
+        String exportedMethodsString = "\"" + String.join("\", \"", exportedMethodsList) + "\"";
```

Comment:
I think this is more intuitive:

        String exportedMethodsString = exportedMethodsList.stream()
            .map(s -> "\"" + s + "\"")
            .collect(Collectors.joining(", "));